tests/browseruse/base_test.py

```
@@ -64,21 +70,49 @@ async def run_task(self, task: str, max_steps: int = 10, controller=None):
+    async def _take_screenshot(self, name: str = "screenshot"):
+        """Take a screenshot using the current browser context."""
+        try:
+            if hasattr(self.browser_session, 'context') and self.browser_session.context:
+                # Get the current page
+                pages = self.browser_session.context.pages
+                if pages:
+                    page = pages[-1]  # Get the most recent page
+                    screenshot_path = self.logs_dir / f"{name}_screenshot.png"
+                    await page.screenshot(path=str(screenshot_path), full_page=True)
+                    print(f"📸 Screenshot saved: {screenshot_path}")
+                    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)